Makefile

```diff
@@ -113,16 +113,27 @@ srpm: .srpm-done
 
 rpm: .rpm-done
 
-ubuntu-trusty:
-	cp packaging/ubuntu-trusty/ecs.conf ecs.conf
-	tar -czf ./amazon-ecs-init_${VERSION}.orig.tar.gz ecs-init ecs.conf scripts README.md
+ARCH:=$(shell uname -m)
+ifeq (${ARCH},x86_64)
+	AGENT_URL=https://s3.amazonaws.com/amazon-ecs-agent/ecs-agent-v${VERSION}.tar
+else ifeq (${ARCH},aarch64)
+	AGENT_URL=https://s3.amazonaws.com/amazon-ecs-agent/ecs-agent-arm64-v${VERSION}.tar
+endif
+
+download-agent:
```

Comment:
This recipe should be declared PHONY, or better yet its name should match that of the downloaded image so it can be used properly as a dependency of downstream targets. If you get the dependencies done right, and want (for example) to build a .rpm and a .deb, make will know that it doesn't need to retrieve the agent multiple times.

Reply:
done